features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64Serializer.kt

```diff
@@ -87,21 +87,24 @@ class Base64Serializer private constructor(
     }
 
     @WorkerThread
-    private fun convertBmpToBase64(bitmap: Bitmap): String {
-        val byteArrayOutputStream = webPImageCompression.compressBitmapToStream(bitmap)
+    private fun convertBmpToBase64(initialBitmap: Bitmap): String {
+        var base64Result = ""
 
-        if (isOverSizeLimit(byteArrayOutputStream.size())) {
-            return ""
-        }
+        val scaledBitmap = drawableUtils.downscaleBitmapToApproxSize(initialBitmap)
```

Comment:
I would add what `approximate` means in this context, maybe it is worth adding the doc / explanation to the method?